src/containers/NFT/NFTHeader/styles.scss

```diff
@@ -0,0 +1,401 @@
+@import '../../shared/css/variables.scss';
+
+.section {
+  margin: 0 32px;
+  @include for-size(tablet-landscape-up) {
+    width: 552px;
+    margin: 0px auto;
+  }
+
+  @include for-size(desktop-up) {
+    width: 1140px;
+  }
+}
+
+table {
+  margin-top: 24px;
+  font-size: 14px;
+
+  .col1 {
+    width: 180px;
+    padding-bottom: 8px;
+    color: $gray-400;
+    text-transform: uppercase;
+    @include semibold;
+  }
+  .col2 {
+    padding-bottom: 8px;
+    color: $white;
+
+    a {
+      color: $green;
+      text-decoration: none;
+    }
+
+    .copy {
+      position: absolute;
+      width: 17px;
+      height: 17px;
+      margin-left: 8.42px;
+    }
+  }
+}
+
+.header-container {
+  .bottom-container {
+    display: flex;
+    flex-direction: column;
+
+    @include for-size(desktop-up) {
+      flex-direction: row;
+    }
+
+    .details {
+      width: 100%;
+      @include for-size(desktop-up) {
+        width: 490px;
+      }
+    }
+
+    .settings {
+      width: 100%;
+      @include for-size(desktop-up) {
+        width: 650px;
+      }
+    }
+
+    .title {
+      padding: 64px 0 18px 0;
+      border-bottom: 1px $gray-800 solid;
+      color: $white;
+      font-size: 24px;
+      @include bold;
+
+      @include for-size(desktop-up) {
+        padding-top: 80px;
+      }
+    }
+  }
+
+  .info-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    margin-top: 68px;
+
+    @include for-size(desktop-up) {
+      flex-direction: row;
+      margin-top: 80px;
+    }
+
+    .values {
+      display: flex;
+      flex-direction: row;
+      justify-content: space-between;
+      margin-bottom: 18px;
+      color: $white;
+
+      @include for-size(desktop-up) {
+        flex-direction: column;
+        margin-bottom: 0;
+      }
+
+      .title {
+        padding-bottom: 4px;
+        margin-bottom: 4.5px;
+        color: $gray-400;
+        font-size: 14px;
+        text-transform: uppercase;
+        @include semibold;
+      }
+      .value {
+        display: flex;
+
+        .value-content {
+          margin: auto;
+          color: #fff;
+          font-family: 'Work Sans', sans-serif;
```

Comment:
constant

Reply:
Actually I think maybe the mockups have work sans being used but literarlly the rest of the site is Space Mono.  This should be done with other work.